src/build.xml

```diff
@@ -17,8 +18,8 @@
 </Target>
 
 <Target Name="RestorePackages">
-  <Exec Command="powershell.exe -ExecutionPolicy ByPass -Command Invoke-WebRequest -Uri https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile $(MSBuildProjectDirectory)\..\tools\Nuget\NuGet.exe" />
-  <Exec Command="$(MSBuildProjectDirectory)\..\tools\Nuget\NuGet.exe restore $(Solution)"/>
+  <Exec Condition="!Exists($(NuGetPath))" Command="powershell.exe -ExecutionPolicy ByPass -Command Invoke-WebRequest -Uri https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile $(NuGetPath)" />
```

Comment:
so the idea here is to only pull nuget if its not already present at the `NuGetPath`?

Reply:
That's correct